language/documentation/hackathon-tutorial/README.md

```diff
@@ -493,30 +493,64 @@ to reuse code since the generic library module already provides a standard imple
+Smart contracts deployed on the blockchain may maniputate high-value assets. As a technique that uses strict mathematical methods to describe behavior and reason correctness of computer systems, formal verification has been used in blockchains to prevent bugs in smart contracts. [The Move prover](https://github.com/diem/diem/tree/main/language/move-prover) is an evolving formal verification tool for smart contracts written in the Move language. The user can specify functional properties of smart contracts using the [Move Specification Language (MSL)](https://github.com/diem/diem/blob/main/language/move-prover/doc/user/spec-lang.md) and then use the prover to automatically check them statically. To illustrate how the prover is used, we add the following code snippet to the BasicCoin.move:
 
-The property specification is written in the [Move Specification Language (MSL)](https://github.com/diem/diem/blob/main/language/move-prover/doc/user/spec-lang.md). Developers can provide pre and post conditions for functions, which include conditions over (mutable) parameters and global memory. Developers can also provide invariants over data structures, as well as the (state-dependent) content of the global memory. Universal and existential quantification both over bounded domains (like the indices of a vector) as well of unbounded domains (like all memory addresses, all integers, etc.) are supported. In this tutorial, we will learn how to define functional properties for methods.
+```
+    spec balance_of {
+        pragma aborts_if_is_strict;
+    }
+```
 
-#### Method `withdraw`
+Informally speaking, the block `spec balance_of {...}` contains the property specification of the method `balance_of`. Let's first run the prover using the command `move package prove` inside the `sources` directory, which outputs the following error information:
 
-The signature of the method `withdraw` is given below:
 ```
-fun withdraw<CoinType>(addr: address, amount: u64) : Coin<CoinType> acquires Balance
+error: abort not covered by any of the `aborts_if` clauses
```

Comment:
The code in `step_7` already contains the aborts_if spec so when users run `move package prove` they won't actually see the error. Maybe it's better to drop this aborts_if from `step_7/BasicCoin/sources/BasicCoin.move` and have them add it themselves?